src/app/features/fee-editor/fee-editor.utils.ts

```diff
@@ -26,18 +26,27 @@ export function formatFeeItem({ fee, feeType, marketData }: FormatFeeItemArgs) {
   const formattedMoneyPadded = txFee?.symbol === 'BTC' && formatMoneyPadded(txFee);
   const formattedMoney = txFee ? formatMoney(txFee) : 'N/A';
 
+  function getCaptionRight() {
+    if (!txFee) return null;
+    return showFeeRate
+      ? `${feeRate} ${feeSymbolToFractionalUnitMap[txFee.symbol]}/vB · ${i18nFormatCurrency(
+          baseCurrencyAmountInQuote(
+            createMoney(Math.ceil(txFee.amount.toNumber()), txFee.symbol),
+            marketData
+          )
+        )}`
+      : `${i18nFormatCurrency(
+          baseCurrencyAmountInQuote(
+            createMoney(Math.ceil(txFee.amount.toNumber()), txFee.symbol),
+            marketData
+          )
+        )}`;
+  }
```

Comment:
I find this logic quite hard to follow, would love it broken down in a more digestible manner

Reply:
Yeah, not great, I think prob better to have two different ones for showing fee or feeRate separately.